Instastories.py

```diff
@@ -203,23 +211,53 @@ def nicks_to_ids(usr_list):
     return ids    
 
 
+def numberOfPeople():
+
+    count = int(input("Please insert the number of people to scrape: "))
+    return count
+
+
+
+def logToHTML():
+    rendered_log = []
+    if not os.path.exists("run_history.log"):
```

Comment:
Just use 'w' mode in the next with open, so it create it if it's missing

Reply:
The 'w' mode creates more problems than this tricky trick